components/js/prophet-wasmstan/package.json

```
@@ -5,7 +5,7 @@
     "Ben Sully <ben.sully@grafana.com>"
   ],
   "description": "Prophet's Stan model, compiled to WASM, for use with the @bsull/augurs library.",
-  "version": "0.1.1",
+  "version": "0.1.2-dev.0",
```

```suggestion
  "version": "0.2.0",
```

Needs a minor version bump due to API incompatibility